service/src/main/resources/application.yml

```diff
@@ -89,7 +89,7 @@ profile:
     managed-app-tenant-id: ${env.azure.managedAppTenantId}
 
     application-offers:
-      fake-terra-app:
+      - name: terra-workspace-dev
```

Comment:
I think `terra-workspace-dev` should become the default offer BPM allows in DEV, see this [thread](https://broadinstitute.slack.com/archives/C02GJ782BAA/p1660665478963609)